resources-dut/strings/strings.xml

```diff
@@ -1,81 +1,81 @@
+	<string id="AppVersionTitle">App Versie</string>
 
-	<string id="ThemeTitle">Theme</string>
-	<string id="HoursColourTitle">Hours Colour</string>
-	<string id="MinutesColourTitle">Minutes Colour</string>
-	<string id="LeftGoalMeterTitle">Left Meter</string>
-	<string id="RightGoalMeterTitle">Right Meter</string>
-	<string id="CaloriesGoalTitle">Calories Goal (1-10,000kCal)</string>
-	<string id="GoalMeterStyleTitle">Meter Style</string>
-	<string id="MeterDigitsStyleTitle">Meter Digits Style</string>
-	<string id="LocalTimeInCityTitle">Add Local Time in City (Beta)</string>
-	<string id="MoveBarStyleTitle">Move Bar Style</string>
-	<string id="HideSecondsTitle">Hide Seconds</string>
-	<string id="HideHoursLeadingZeroTitle">Hide Hours Leading Zero</string>
+	<string id="ThemeTitle">Thema</string>
+	<string id="HoursColourTitle">Uren Kleur</string>
+	<string id="MinutesColourTitle">Minuten kleur</string>
+	<string id="LeftGoalMeterTitle">Linker Meter</string>
+	<string id="RightGoalMeterTitle">Rechter Meter</string>
+	<string id="CaloriesGoalTitle">Calorien Doel (1-10,000kCal)</string>
+	<string id="GoalMeterStyleTitle">Meter Stijl</string>
+	<string id="MeterDigitsStyleTitle">Meter getallen Stijl</string>
+	<string id="LocalTimeInCityTitle">Voeg Lokale Tijd In Stad Toe (Beta)</string>
+	<string id="MoveBarStyleTitle">Beweeg Balk Stijl</string>
+	<string id="HideSecondsTitle">Verberg Seconden</string>
+	<string id="HideHoursLeadingZeroTitle">Verberg Eerste Nul Bij Uuren</string>
 
-	<string id="FieldCountTitle">Number Of Data Fields</string>
-	<string id="DataField1Title">Data Field 1</string>
-	<string id="DataField2Title">Data Field 2</string>
-	<string id="DataField3Title">Data Field 3</string>
+	<string id="FieldCountTitle">Hoeveelheid Datavelden</string>
+	<string id="DataField1Title">Data Veld 1</string>
+	<string id="DataField2Title">Data Veld 2</string>
+	<string id="DataField3Title">Data Veld 3</string>
 
-	<string id="IndicatorCountTitle">Number Of Indicators</string>
+	<string id="IndicatorCountTitle">Hoeveelheid Indicatoren</string>
 	<string id="Indicator1Title">Indicator 1</string>
 	<string id="Indicator2Title">Indicator 2</string>
 	<string id="Indicator3Title">Indicator 3</string>
 
-	<string id="FromTheme">(From Theme)</string>
+	<string id="FromTheme">(Van Thema)</string>
 	<string id="MonoHighlight">Mono Highlight</string>
 	<string id="Mono">Mono</string>
 
-	<string id="AllSegmentsMerged">All Segments (Merged)</string>
-	<string id="FilledSegmentsMerged">Filled Segments (Merged)</string>
-	<string id="AllSegments">All Segments</string>
-	<string id="FilledSegments">Filled Segments</string>
-	<string id="Hidden">Hidden</string>
-	<string id="CurrentTarget">Current/Target</string>
-	<string id="Current">Current</string>
+	<string id="AllSegmentsMerged">Alle Segmenten (Samen)</string>
+	<string id="FilledSegmentsMerged">Gevulde Segments (Samen)</string>
+	<string id="AllSegments">Alle Segments</string>
```

Comment:
@DRG-developer Many thanks for doing the translation, and sorry for the delay. I'd like to get this released this week. Just wanted to double-check: should "Alle Segments" be "Alle Segmenten"? Or is it OK as is? And same for "Gevulde Segments".

Reply:
Yes, good catch! it should be "Alle segmenten" and "Gevulde segmenten" indeed!

Reply:
OK, thank you! I can correct those after merge 👍